app/jobs/appointment_notification/worker.rb

```diff
@@ -6,14 +6,22 @@ class AppointmentNotification::Worker
 
   DEFAULT_LOCALE = :en
 
+  def metrics
+    @metrics ||= Metrics.with_object(self)
+  end
+
   def perform(appointment_id, communication_type, locale = nil)
     appointment = Appointment.find_by(id: appointment_id)
     unless appointment
+      metrics.increment(:skipped)
```

Comment:
Consider `skipped.missing_appointment`, and `skipped.previously_communicated` below?